_data/sites.json

```diff
@@ -11611,6 +11622,17 @@
         ]
     },
 
+    {
+        "name": "Piazza",
+        "url": "https://support.piazza.com/support/solutions/articles/48000616709-student-delete-your-account",
+        "difficulty": "hard",
+        "notes": "E-mail help@piazza.com and ask to close your account. They will reply asking you to confirm, after which point you can acknowledge and it will be removed.",
```

Comment:
```suggestion
        "notes": "E-mail them and ask to close your account. They will reply asking you to confirm, after which point you can acknowledge and it will be removed.",
```